### PR TITLE
OP-1064 Make list of exams laboratory parametric and pageable

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OP-1064.

### CODE REVIEW

**Review by mwithi — DISMISSED** [review]
LGTM. Thanks a lot!

File: src/main/java/org/isf/lab/manager/LabManager.java [hunk 1]
```diff
@@ -117,6 +117,19 @@ protected void validateLaboratory(Laboratory laboratory) throws OHDataValidation
...
+	/**
+	 * Return the list of exams ({@link Laboratory}s) divided by pages.
+	 * 
+	 * @param oneWeek
+	 * @param pageNo
+	 * @param pageSize
+	 * @return the list of {@link Laboratory}s. It could be <code>empty</code>.
+	 * @throws OHServiceException
+	 */
+	public List<Laboratory> getLaboratory(boolean onWeek, int pageNo, int pageSize) throws OHServiceException {
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/lab/manager/LabManager.java:129]: The Javadoc has the variable as `oneWeek` (see line 123) and the method signature uses `onWeek`; pick one

File: src/main/java/org/isf/lab/service/LabIoOperations.java [hunk 2]
```diff
@@ -76,6 +79,27 @@ public List<LaboratoryRow> getLabRow(Integer code) throws OHServiceException {
...
+	/**
+	 * Return the list of exams ({@link Laboratory}s) divided by pages.
+	 * 
+	 * @param oneWeek
+	 * @param pageNo
+	 * @param pageSize
+	 * @return the list of {@link Laboratory}s (could be empty)
+	 * @throws OHServiceException
+	 */
+	public List<Laboratory> getLaboratory(boolean onWeek, int pageNo, int pageSize) throws OHServiceException {
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/lab/service/LabIoOperations.java:91]: The Javadoc has the variable as  `oneWeek` (see line 85) and the method signature uses `onWeek`; pick one

File: src/main/java/org/isf/lab/service/LabIoOperations.java [hunk 3]
```diff
@@ -76,6 +79,27 @@ public List<LaboratoryRow> getLabRow(Integer code) throws OHServiceException {
...
+		LocalDateTime time2 = TimeTools.getDateToday24();
+		LocalDateTime time1 = time2.minusWeeks(1);
+		Pageable pageable = PageRequest.of(pageNo, pageSize);
+		if (onWeek) {
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/lab/service/LabIoOperations.java:95]: Why aren't the 2 time calculations inside the `if` block? That is the only place `time2` and `time1` are used.

**Review by dbmalkovsky — COMMENTED** [review]
I have 3 comments before I will approve this PR; thanks

**Review by dbmalkovsky — APPROVED** [review]
LGTM

**Pull request merged**